geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+
+C = O     1.162(5)
+C = C     1.3136(56) 
+C   H     1.0795(17) 
+H   C = C 118.9(30)
+
+# Bond lengths: rm (angstroms)
+# Bond angles:  thetam (degrees)
+
+C = O     1.1617(4)
+C = C     1.3149(6) 
+C   H     1.1617(4)
+H   C = C 118.72(2)
+
+# Bond lengths: re^se (angstroms)
+# Bond angles:  thetae^se (degrees)
+
+C = O     1.1607(6)
+C = C     1.3122(6)
+C   H     1.0763(1)
+H   C = C 119.115(11)
+
+# Bond lengths: rh1 (angstroms)
+# Bond angles:  thetah1 (degrees)
+
+C = O     1.154(3)
+C = C     1.323(2)
+C   H     1.085(1)
+H   C = C 119.4(5)
+H   C   H 121.3(10)
+
+# C = O 1.154(3), C = C 1.323(2), C H 1.085(1) and H C = C 119.4(5) constrained to the value from MP2/6-311++G** calculation. 
+# H C H 121.3(10) dependent parameter.
+
+NAME = C2H3As 2009
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Demaison J, Møllendal H., Guillemin JC (2009) Equilibrium CAs and CSb bond lengths. J Mol Struct 930(1-
+3):21-25
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C ≡ As  1.6585(2)
+C   C   1.4634(2)
+C   H   1.0910(3)
+H   C C 110.575(10)
+
+NAME = C2H5N 
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Liu X, Xu Y (2011) Infrared and microwave spectra of the acetylene-ammonia and carbonyl sulfide-ammonia
+complexes: a comparative study of a weak C-H⋅⋅⋅N hydrogen bond and an S⋅⋅⋅N bond. Phys Chem Chem Phys
+13(31):14235-14242
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+H...N    2.3980(7)
+NH3 HCCH 4.0594(6)
+α        23.29(2)
+
+# α: Angular oscillation of ammonia subunit. 
+
+NAME = C2N2 2011
+# Point Group: Dooh 
+# Shape: Linear
+# Bond lengths: re (angstroms)
+# Source paper: Maki AG (2011) High-resolution infrared spectrum of cyanogen. J Mol Spectrosc 269(2):166-174
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C   C 1.38109(60)
+C ≡ N 1.15976(40)
+
+NAME = C2N2S 2013
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Kisiel Z, Winnewisser M, Winnewisser BP, DeLucia FC, Tokaryk DW, Billinghurst BE (2013) Far-infrared
+spectrum of S(CN)2 measured with synchrotron radiation: global analysis of the available high-resolution
+spectroscopic data. J Phys Chem A 117(50):13815-13824
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+S   C     1.698(6)
+C ≡ N     1.160(9) 
+C   S C   98.6(5)
+S   C ≡ N 175.4(11)
+
+NAME = C2OS3 2010
+# Point Group: Cs
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Norooz Oliaee J, Mivehvar F, Dehghany M, Moazzen-Ahmadi N (2010) Geometric isomerism in the OCS-CS2
+complex: Observation of a cross-shaped isomer. J Phys Chem A 114(27):7311-7314
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+S2C2O  SC1S        3.5553(8)
+C(1)...C(2) = S(2) 104.82(22)
```

Comment:
S2C2O is a pentaatomic molecule, but the books is only giving us a triatomic molecule here.

These labels in parentheses also won't help without the diagram. 

Also, all four bonds are double bonds, so C(2) = S(2) won't tell us to which pair of C and S atoms is being referred. 

Maybe this will work:
```
COS(1) CS2         3.5553(8)
C          C      S(1) 104.82(22)
```
Or even better, I think the CCO angle is just 180 minus the CCS(1) angle, so we could do:

```
COS CS2         3.5553(8)
C       C      O    180-104.82(22)
```
But we might need to label one of the C atoms because there's two possible CCO angles. Only one of them avoids jumping from molecule 1 to molecule 2 and back to molecule 1 though.